.github/workflows/ci.yml

```
@@ -83,13 +83,164 @@ jobs:
+
+  test-summary:
+    runs-on: ubuntu-latest
+    name: Test Summary
+    if: always()
+    needs:
+      - checks
+      - lsp-integration-tests
+      - neovim-integration-tests
+    steps:
+    - name: Generate test summary
+      run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2129:style:1:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)